### PR TITLE
getOrchestrator docstring fix

### PR DESCRIPTION
**Pull request by PavelZaytsev**
## Overview

Description:

Why should this be merged: 
Fix description to better reflect what the function is doing. 
Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java [hunk 1]
```diff
@@ -46,8 +46,8 @@
     protected abstract UUID sendRequest(ManagementClient client) throws TimeoutException;
 
     /**
-     * Select an orchestrator and return a client. Orchestrator's that
-     * are on unresponsive nodes and the affected endpoint by the workflow
+     * Select an orchestrator and return a client. Orchestrator
+     * located on the affected by the workflow endpoint
```
Comment (zalokhan) [inline runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java]: Orchestrator located on the affected node and nodes not responding to pings will not be selected.

Comment (PavelZaytsev) [reply runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java]: sounds good

File: runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java [hunk 2]
```diff
@@ -46,9 +46,9 @@
     protected abstract UUID sendRequest(ManagementClient client) throws TimeoutException;
 
     /**
-     * Select an orchestrator and return a client. Orchestrator's that
-     * are on unresponsive nodes and the affected endpoint by the workflow
-     * will not be selected. The layout might not reflect the state
+     * Select an orchestrator and return a client. Orchestrator
+     * located on the affected by the workflow endpoint or nodes not responding
+     * to pings will not be selected. The layout might not reflect the state
```
Comment (Maithem) [inline runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java]: Consider rephrasing the whole sentence.
" Select an orchestrator that responds to pings and is not on the same node affected by the workflow. "

Comment (vjeko) [reply runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java]: "located on the affected by the workflow" reads a bit unnatural. I also suggest the re-wording.

Comment (PavelZaytsev) [reply runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java]: fixed

**Pull request merged**